File: app/src/main/java/be/vrt/poc/playpausestopanimatedvector/MainActivity.java
package be.vrt.poc.playpausestopanimatedvector;

import android.graphics.drawable.Animatable;
import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);/*
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);*/
    }

    public void startAnimation(View view) {
        final ImageView imageView = (ImageView) view;
        final Drawable drawable = imageView.getDrawable();
        if (drawable instanceof Animatable) {
            ((Animatable) drawable).start();
        }
    }
}

File: app/src/main/kotlin/be/rijckaert/tim/animatedvector/MainActivity.kt
package be.rijckaert.tim.animatedvector

import android.graphics.drawable.Animatable
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.widget.ImageView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun startAnimation(view: View) {
        val imageView = view as ImageView
        val drawable = imageView.drawable
        if (drawable is Animatable) {
            drawable.start()
        }
    }
}